### PR TITLE
#68 Fix templateSchema

### LINKED ISSUE #68: [SCS-MultiApi-Plugin] [OpenApi] Fix TemplateSchema not to use a specific dependency
As a developer, I want to review the builder's creation of a list type attribute in the TemplateSchema.ftlh so the user is not obligated to use the CollectionUtils dependency.

### PR DESCRIPTION
**Pull request by jmejutovazquez**
Closes #68

### CODE REVIEW

File: src/test/resources/net/coru/api/generator/multiapi/integration/test/ScsMultiapiGenerationTest/testScsMultiapiGeneration/assets/ApiErrorDTO.java [hunk 1]
```diff
@@ -21,6 +21,7 @@ public static class ApiErrorDTOBuilder {
     private String message;
     private Integer code;
 
+
```
Comment (jemacineiras) [inline src/test/resources/net/coru/api/generator/multiapi/integration/test/ScsMultiapiGenerationTest/testScsMultiapiGeneration/assets/ApiErrorDTO.java:24]: Not needed this empty line

File: src/test/resources/net/coru/api/generator/multiapi/integration/test/ScsMultiapiGenerationTest/testScsMultiapiGeneration/assets/ApiTestDTO.java [hunk 2]
```diff
@@ -21,6 +21,7 @@ public static class ApiTestDTOBuilder {
     private String name;
     private Integer id;
 
+
```
Comment (jemacineiras) [inline src/test/resources/net/coru/api/generator/multiapi/integration/test/ScsMultiapiGenerationTest/testScsMultiapiGeneration/assets/ApiTestDTO.java:24]: This empty line is not need

File: src/test/resources/net/coru/api/generator/multiapi/integration/test/ScsMultiapiGenerationTest/testScsMultiapiGeneration/assets/ApiTestInfoDTO.java [hunk 3]
```diff
@@ -24,19 +23,20 @@ public static class ApiTestInfoDTOBuilder {
     private String testName;
     private List<String> testers = new ArrayList<String>();
 
+
```
Comment (jemacineiras) [inline src/test/resources/net/coru/api/generator/multiapi/integration/test/ScsMultiapiGenerationTest/testScsMultiapiGeneration/assets/ApiTestInfoDTO.java:26]: Wrong Empty Line

File: pom.xml [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
   <groupId>net.coru</groupId>
   <artifactId>scs-multiapi-maven-plugin</artifactId>
-  <version>1.5.2</version>
+  <version>1.5.5</version>
```
Comment (jemacineiras) [inline pom.xml:7]: Wrong version number

**Review by jemacineiras — CHANGES_REQUESTED** [review]

**Review by jemacineiras — APPROVED** [review]

**Review by fcampostato — APPROVED** [review]

**Review by Rivarsal — APPROVED** [review]

**Pull request merged**